vocab/properties.json

```diff
@@ -130,38 +168,51 @@
         "label": "Acknowledgement",
         "labelForReverseLink": "Is acknowledgement of",
         "name": "acknowledgement",
+        "nameForReverseLink": "isAcknowledgementOf",
         "sameAs": null,
         "schemas": [
             "core/v0/miscellaneous/funding",
             "core/v3/miscellaneous/funding",
             "core/v4/miscellaneous/funding"
-        ]
+        ],
+        "formating": {
+            "formattedText": true,
+            "multilineText": true
+        }
     },
     "https://openminds.ebrains.eu/vocab/addExistingTerminology": {
         "description": "Reference to an existing terminology (distinct class to group related terms).",
         "label": "Add existing terminology",
-        "labelForReverseLink": "Is add existing terminology of",
+        "labelForReverseLink": "Suggested in",
         "linkedTypes": [
             "https://openminds.ebrains.eu/controlledTerms/Terminology"
         ],
         "name": "addExistingTerminology",
+        "nameForReverseLink": "suggestedIn",
         "schemas": [
             "controlledTerms/v1/termSuggestion"
         ]
     },
     "https://openminds.ebrains.eu/vocab/additionalInformation": {
         "description": null,
         "label": "Additional information",
+        "labelForReverseLink": "Are additional information for",
```

Comment:
I think this should be "Is additional information for"

Reply:
Actually I think this shouldn't be anything. there are no linked Types that means no reversed link either